README.md

```diff
@@ -31,27 +32,29 @@ pip install -r requirements.txt
 ```
 
 # USAGE
-You can analyze multiple folders (using the same scenario) with one execution, simply provide the paths to all the SMAC3-results in `--folders`.
+You can analyze multiple folders (that are generated with the same scenario) for the analysis, simply provide the paths to all the individual SMAC3-results in `--folders`.
 
 Commandline arguments:
-- --folders: path(s) to folder(s) containing the SMAC3-output
+- `--folders`: path(s) to folder(s) containing the SMAC3-output (works with
+  `output/run_*`)
 
 Optional:
-- --output: where to save the CAVE-output
-- --ta_exec_dir: target algorithm execution directory, this should be a path to
+- `--output`: where to save the CAVE-output
+- `--ta_exec_dir`: target algorithm execution directory, this should be a path to
   the directory from which SMAC was run initially. used to find instance-files and
   if necessary execute the `algo`-parameter of the SMAC-scenario (DEFAULT:
   current working directory)
-- --param_importance: calculating parameter importance is expensive, so you can
+- `--param_importance`: calculating parameter importance is expensive, so you can
   specify which plots you desire: `ablation`, `forward_selection`, `fanova`
   and/or `incneighbor`.
```

Comment:
Incneighbor is now called LPI/local parameter importance

Reply:
Currently those are still the actual arguments passed to pimp, which still uses `incneighbor`. Will make them more verbose soon.

Reply:
I'll handle that accordingly in PIMP